.github/workflows/release.yaml

```diff
@@ -47,14 +47,19 @@ jobs:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
           CR_SKIP_EXISTING: "true"
 
+      - name: Login to GHCR
+        uses: docker/login-action@v3.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Push charts to GHCR
         run: |
           shopt -s nullglob
-          for pkg in .cr-release-packages/*; do
+          for pkg in .cr-release-packages/*.tgz; do
             if [ -z "${pkg:-}" ]; then
               break
             fi
-            if ! helm push "${pkg}" "oci://ghcr.io/${GITHUB_REPOSITORY_OWNER}/charts"; then
-              echo '::warning:: helm push failed!'
-            fi
+            helm push "${pkg}" "oci://ghcr.io/${GITHUB_REPOSITORY_OWNER}/charts"
```

Comment:
Just to be in sync with the classic workflow
```bash
helm repo add grafana https://grafana.github.io/helm-charts
```

Shall we change this also to oci://ghcr.io/${GITHUB_REPOSITORY_OWNER}/ **helm-**charts

Reply:
I would say, the "world" is using "charts", including prometheus-community, fluxcd, velero.

Reply:
yes but helm-charts already exists for other Grafana-labs projects
https://github.com/grafana/grafana-operator/pkgs/container/helm-charts%2Fgrafana-operator

Reply:
Then it makes sense!